experiments/regex/dynamic_re_inclusive_over.js

```diff
@@ -0,0 +1,164 @@
+export { generateInclusiveOverRegex };
+
+/**
+ * Generates a regex that maches all numbers from the next factor i.e. 100 -> [1-9][0-9][0-9][0-9]+
```

Comment:
det skal vel være * frem for +??

Reply:
Nej

Reply:
Det pattern gør det kun for næste faktor, så 100'er -> 1000'er.

Det andet håndteres med round_numbers_pattern + 'pattern'